python/pyspark/sql/utils.py

```diff
@@ -61,12 +67,48 @@ def deco(*a, **kw):
                                              e.java_exception.getStackTrace()))
             if s.startswith('org.apache.spark.sql.AnalysisException: '):
                 raise AnalysisException(s.split(': ', 1)[1], stackTrace)
+            if s.startswith(
+                    'org.apache.spark.sql.catalyst.analysis.DatabaseAlreadyExistsException: '):
```

Comment:
can you just do `startsWith('org.apache.spark.sql.catalyst.analysis')`? It's a lot less code to express the same thing.